app/build.gradle

```diff
@@ -34,12 +34,28 @@ android {
 
 dependencies {
 
+    implementation "io.github.microutils:kotlin-logging:2.0.6"
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.5.0'
     implementation 'androidx.appcompat:appcompat:1.3.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'androidx.recyclerview:recyclerview:1.2.0'
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+
+    //Retrofit, OkHttp and RxJava
+    implementation 'com.squareup.retrofit2:retrofit'
+    implementation 'io.reactivex.rxjava3:rxjava:3.0.1'
+    implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
+    implementation "com.github.akarnokd:rxjava3-retrofit-adapter:3.0.0"
+    implementation 'com.squareup.retrofit2:converter-gson:2.7.2'
+    implementation 'com.squareup.okhttp3:logging-interceptor:3.14.9'
+
+    // Recyclerview
+    implementation 'androidx.recyclerview:recyclerview:1.1.0'
```

Comment:
For some reason, you import the same lib twice but different versions? Check line 43 too?